pages/applications/edit/[id]/[person]/index.tsx

```diff
@@ -94,12 +98,12 @@ export default function EditApplicant({ user, data }: PageProps): JSX.Element {
 
     setIsSubmitted(true);
   };
-
+  /* eslint-disable react/jsx-no-useless-fragment */
   return (
     <>
       {data.id ? (
         <MainApplicantForm
-          isEditing={true}
+          isEditing={true} //eslint-disable-line
```

Comment:
Out of interest, what error was throwing here?

Reply:
this is where it changes it from `isEditing={true}` to`isEditing` which feels a bit dangerous without further investigation

Reply:
Oh that's right - that's the one from yesterday.